### PR TITLE
Rework the qualification points page to use Django form handling (closes #97)

### LINKED ISSUE #97: Points input fields are not filled when opening the qualification results entry page
It seems that they started getting formated using Polish locale with `,` instead of `.`, did anybody change any settings related to that?

```html
<input type=number step=0.1
  value="6,0"
  class="points-input"
  data-id="1234">
```

### PR DESCRIPTION
**Pull request by krzys-h**
This fixes problems caused by recent changes to internationalization settings. Closes #97.

I also improved the page layout a bit:
image

---
Additionally, I fixed two minor issues related to POST data handling that appeared on Sentry a while ago (or.. err.. actually somehow they never made it to Sentry and I only got an email from Django itself about it, whaaat? @cytadela8) This one:
```
Internal Server Error: /register/

MultiValueDictKeyError at /register/
"'workshop_name'"
```

### CODE REVIEW

File: templates/workshopparticipants.html [hunk 1]
```diff
@@ -13,57 +14,61 @@ <h1>{{ workshop.title }} &dash; uczestnicy</h1>
...
         {% endfor %}
         </tbody>
       </table>
       <div class="containter-fluid">
-        Maile: <input type="text"
-                      value="{% for workshop_participant in workshop_participants %}{{ workshop_participant.participant.user.email }} {% endfor %}"
-                      id="participants_emails">
+        <div class="form-group">
+          <label for="participants_emails">Maile:</label>
+          <input id="participants_emails" class="form-control" value="{% for workshop_participant in workshop_participants %}{{ workshop_participant.participant.user.email }}, {% endfor %}" />
```
Comment (cytadela8) [inline templates/workshopparticipants.html]: @grzegorz225 This is email so in no way can this be exploited, but I wonder if we see here a possibility of exploits for some Django apps (if you control a html tag parameter can you do something like: `" onclick="alert(1);`?)

File: wwwapp/views.py [hunk 2]
```diff
@@ -314,37 +314,29 @@ def workshop_participants_view(request, name):
...
-    except (ValidationError, ValueError) as e:
-        print(e)
-        return JsonResponse({'error': 'Niepoprawny format liczby'})
-    except Exception as e:
-        print(e)
-        raise e
-    workshop_participant = WorkshopParticipant.objects.get(id=workshop_participant.id)
-
-    return JsonResponse({'points': str(workshop_participant.qualification_result),
+    form = WorkshopParticipantPointsForm(request.POST, instance=workshop_participant)
```
Comment (cytadela8) [inline wwwapp/views.py]: NICE

Comment (cytadela8) [reply wwwapp/views.py]: We should handle more things with forms...

File: templates/workshopparticipants.html (hunk 1 above)
Comment (krzys-h) [reply templates/workshopparticipants.html]: You can't, unless you specifically tell Django to disable autoescaping

https://docs.djangoproject.com/en/3.0/ref/templates/language/#automatic-html-escaping

**Comment by krzys-h** [comment]
> I found a weird (kinda, although also kinda justified) Firefox behaviour. If you change some points refreshing the website won't make you loose the change, but (obviously) the save button is disabled.

Wow, I had no idea Firefox did that! Chrome doesn't... Apparently, [setting autocomplete to off disables this](https://stackoverflow.com/questions/7377301/firefox-keeps-form-data-on-reload)

File: templates/workshopparticipants.html (hunk 1 above)
Comment (cytadela8) [reply templates/workshopparticipants.html]: You are right `"` is escaped.

**Pull request merged**